Spelling, Formating, and other minor changes

mcdonnelldean opened the pull request:
@mcollina Some changes on my _night off_ from coding :D Just don't tell the other half :D

mcdonnelldean committed:
Spelling, Formating, and other minor changes

mcollina commented on README.md:
```diff
@@ -250,58 +262,56 @@ Connects this HyperEmitter with another one which we call a peer. Peers can exis
 
-##### _callback_
-An optional function to be called when close has completed.
+ * #### _callback_
```
I'm not sure why, but this does not render well in Github.

mcdonnelldean replied:
@mcollina Specifically the size? They looked a lot smaller in my markdown preview, let me make them a smaller.

mcollina commented:
BTW, I gave you the commit bit, you can send PRs right from this repo ;).

mcdonnelldean commented:
@mcollina Yeah I seen that. I do like to maintain my own fork though, it just means any silly pushes to origin on my part can be nuked by reforking :D

mcdonnelldean committed:
Smaller Arg Headers

mcdonnelldean committed:
Header parsing fix

Looks like the markdown parser gets confused depending on line placement. All args should look right now.

mcdonnelldean commented:
@mcollina That was really odd! A hard space fixed the issue. I made the args smaller as well, they didn't look that big in my previewer!
